test/dsp_test.exs

```diff
@@ -0,0 +1,81 @@
+defmodule Test.Dsp do
+  use ExUnit.Case, async: true
```

Comment:
How exactly do these run concurrently? I understand that you start a `Dsp` GenServer in each test, and you use `Req.Test.allow/3` to seemingly direct requests from the calling pid to the started GenServer, but don't you use a singleton, named ETS table in Dsp? Successful keyring refreshes in concurrent tests will likely overwrite one another.

It might not be an issue right now, but it could make for some hair-pulling debugging.

Reply:
That's a good point. I know it's not happening since attempting to start the Dsp server twice raises an exception since the table already exists. I need to think about this.

Reply:
I originally set `async: true` because otherwise I had some strange failures related to `Req.Test`. I believe this was before I figured how to use `Req.Test.allow/3`. Now, it worked because no other test uses the `Dsp` server, but you're right, as soon as it's used in other tests, it will cause exceptions. I removed the `async` option.